src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
We need to meet over this.  Users do not have hte context to understnd what this means and what the result of their choice will be.  Furthermore, the decision cannot be made without fully understanding what impact this has on our development costs.  Right now, for example, i don't believ ehte IDE has the bandwdith to ensure that all our features will work properly if the system transitions during a VS session from being out of proc to in proc.  Please do not make the feature more complex until we have decided that we can sign up for the work fallout of it.